### ISSUE TITLE
Comments in record type definitions not working

### ISSUE THREAD

**Issue by untoldwind** [untoldwind/KontrolSystem2#50]
So I've been trying to build stuff, and my normal response to problems is to prune the code. But ...

```
// this is a comment, right?

pub type NumStr = (
    n: float,   // this comment is a syntax error.
    s: string
)
```
Why is the comment on the "n:" line causing the error
```
ERROR: [comment.to2(4, 13)] Parsing

    comment.to2(4, 13): expected ')'
```
Intended? Unintuitive ...

**Comment by untoldwind** [comment]
Should be fix in 0.2.1

**Comment by untoldwind** [comment]
Closing this for now as it does not have reappeared yet
